packages/blockchain/src/consensus/clique.ts

```diff
@@ -98,8 +97,23 @@ export class CliqueConsensus implements Consensus {
    */
   public _cliqueLatestBlockSigners: CliqueLatestBlockSigners = []
 
-  constructor() {
+  private debug?: Debugger
+
+  constructor(debug: boolean = false) {
```

Comment:
Ah, that's a bit of a special case, generally good catch that we are not guarding at all the debug calls here for the consensus implementations.

So: we can't add single constructor options here, since all consensus implementations should adhere to the same interface.

So the `debug` option should be added to `ConsensusOptions` in `interface.ts` in this case, and then a local debug guard should be initialized in the `setup()` method (I guess clique-only), since this method is taking in the `ConsensusOptions` as we have implemented it.